build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlinVersion = "1.4.10"
+    ext.kotlinVersion = "1.6.21"
```

Comment:
If this was to be merged, this should be much more thoroughly tested, but it isn't for now. This is needed by some of the testing dependencies